lua/gx/init.lua

```diff
@@ -74,6 +97,7 @@ local function with_defaults(options)
     handlers = options.handlers or {},
     handler_options = {
       search_engine = options.handler_options.search_engine or "google",
+      select_for_search = options.handler_options.select_for_search or false,
```

Comment:
I think I'd separate this behaviour for normal and visual mode.
I'd want to not open search if I hit `gx` in visual mode, but have it give me search in the popup even if another handler matches the visual selection as well.